DO_OPENAPI_COMMIT_SHA.txt

```diff
@@ -1 +1 @@
-47d88fb
+47d88fb and 99df010
```

Comment:
Is this actually valid to have two commits here? 

It breaks the assumptions in the script added here: https://github.com/digitalocean/pydo/pull/134